packages/zoe/test/unitTests/contracts/loan/helpers.js

```diff
@@ -42,7 +42,7 @@ export const checkPayout = async (
 
 /**
  * @param {import("ava").ExecutionContext<unknown>} t
- * @param {ZoeService} zoe
+ * @param {ERef<ZoeService>} zoe
```

Comment:
Just curious: Why did this type check before you fixed it?

Reply:
We don't have typechecking on Zoe test files enforced yet. I turned it on temporarily to help with transitioning from the deprecated ERTP features, then turned it off again because there was much more (unrelated) things that needed fixing.